woocommerce/woocommerce-ios
# Issue/receips hot fix

Closes #4236 

## Changes
* Trigger the "Print receipt" action in the success modal presented when a payment is completed after the modal has been dismissed.

## How to test
* Collect payment for an eligible order
* Tap "print receipt" after the payment has been collected. Notice if the receipt can be printed.

Update release notes:
